### DISCUSSION TITLE
Removal of settings (enable_meta, enable_https_verification, ampm)

### DISCUSSION THREAD

**Discussion by thezoggy** [sabnzbd/sabnzbd#2182] [General]
Got the warning about wanting to removing these options that I have changed:

- enable_meta
config > switches - Use tags from indexer
I have it unchecked. I dont recall changing this one so its prob really old that I did it and cant really see a reason why. So no complaints on this one.

- enable_https_verification
config > general > HTTPS certificate verification
I have it unchecked. at one point I had a local server that I used to pull RSS feed from, and since it had self signed cert I had to uncheck this to make sab happy.. while I don't need this option (stopped using RSS feed) I could see others needing it.

- ampm
config > specials > ampm
I have it unchecked. I'm pretty sure I had this unchecked for older skins.. but I do see it is still used (time_format) for the RSS page and Emailer...  So until those areas are updated to reflect how time is done everywhere it prob should stay. Honestly we might want to adopt how sonarr/radarr offers up date/time options to be a bit more user friendly:
![firefox_2022-05-19_11-54-45](https://user-images.githubusercontent.com/500882/169355741-f2cef1d3-6560-4eb1-85fa-8bbe37f509de.png)

**Comment by Safihre** [comment]
Thanks for the feedback. Based on this and feedback of others I might move stuff to Special instead of fully removing it.

**Comment by GJvdS** [comment]
enable_https_verification

Please move this to Special as you suggested. My configuration is behind a firewall in a home lan setup so I have no need for a certificate. I do not own an official  domain except my internal domain so LetsEncrypt is no option. For me a Self Signed Certificate is not worth the effort because you need to install the CA everywhere you use the certificate.

**Reply by Safihre** [reply]
Just to be sure: this setting is only for _outgoing_ connections. So when we connect to RSS feeds or download NZB's from URLs. So does this apply to your situation?

**Comment by GJvdS** [comment]
Excuse me Safire, my bad.
I main "Enable HTTPS" in my situation. As long as that one is staying I have no issues.

**Reply by Safihre** [reply]
We're keeping that one for sure! Indeed the name of the variable is confusing.

**Comment by CollinChaffin** [comment]
I need to set enable_https_verification to disabled due to one of my indexers is using a cert that does not match the expected hostname.

**Reply by sanderjo** [reply]
Oh? Which indexer is that?

**Comment by Safihre** [comment]
I removed `enable_https_verification` from the list. Seems it is needed.

**Comment by CyberBlaed** [comment]
Used this setting it seems and devs wanted to know why 'AMPM' and thought it was times for the scheduler, seems not the case and cannot find out what it does.
So I really don't know, posting here incase it is used elsewhere in the app and I have not noticed.

**Comment by a2zvargas** [comment]
I have "AMPM" enabled, but I dont know why.  I dont remember turning it on, and I'm not even sure what it does.  😄

**Reply by Slikkster** [reply]
Same here.

**Reply by TheKorn2** [reply]
Dit-oh

**Comment by sunnyd24** [comment]
"ampm" is enabled for me too. The description in help wiki reads "Display ETA in AM/PM notation, only applicable to some locations in older skins."

I prefer ETA in AM/PM notation rather than military time. Is this not required for displaying ETA with AM/PM or is this overridden by "Dates" setting as per image in first post?

**Reply by gibxxi** [reply]
As a UK user, I can say that (in the main) 24hr clock is not ubiquitous here, and some of us literally hate it. I too would prefer this option to remain available. Keep it in the 'Specials' section, but keep it in some form or another please.

Dan / Gib.

**Reply by Safihre** [reply]
Note that this is not the display in the main interface, only for example in the RSS feed display.

**Comment by junkit** [comment]
I got a message about my osx_speed and ampm settings.  I am not using OSX, so I'm not sure how that got changed from default.  As far as ampm, I'm not sure why I changed that from default, other than I changed it during some troubleshooting and forgot to switch it back.  I have no problem with these setting being removed.  Thanks for all the hard work!

**Comment by abatchx** [comment]
ampm unchecked in my instance. I'm almost certain that I never changed this - so I'm guessing its default. Not overly fussed about its removal / update.

**Comment by garyblythe** [comment]
Received the warning about enable_meta recently. My current configuration doesn't use the switch and I can't recall ever having set it, except as a test during some fit of pique over issues with NZB payload renaming... No issues with its removal.

**Comment by Sausages69** [comment]
If the time is going to be displayed as 24hr clock, then I'd prefer it's not removed.
